Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -954,20 +954,21 @@ int RenderBox::horizontalScrollbarHeight() const
     return includeHorizontalScrollbarSize() ? scrollableArea->horizontalScrollbarHeight(IgnoreOverlayScrollbarSize, isHorizontalWritingMode()) : 0;
 }
 
+static bool shouldShowScrollbarGutter(ScrollbarGutter gutter, bool hasVisibleOverflow, Overflow overflow)
```

Comment:
This could be a lambda function.

It's a bit odd to say "show" for something that's just a gap.

Reply:
Changed to lambda. Changed the name to `shouldIncludeScrollbarGutter`.